SDWebImage/UIImage+WebP.m

```diff
@@ -43,21 +43,15 @@ + (nullable UIImage *)sd_imageWithWebPData:(nullable NSData *)data {
         return nil;
     }
     
-    uint32_t flags = WebPDemuxGetI(demuxer, WEBP_FF_FORMAT_FLAGS);
-    if (!(flags & ANIMATION_FLAG)) {
-        // for static single webp image
-        UIImage *staticImage = [self sd_rawWebpImageWithData:webpData];
-        WebPDemuxDelete(demuxer);
-        return staticImage;
-    }
-    
     WebPIterator iter;
```

Comment:
Create Iterator here is useless for static image(Since it will be decoded using `sd_rawWebpImageWithData`